ietf/submit/tests.py

```diff
@@ -1142,7 +1141,7 @@ def test_submit_new_logged_in_with_extresources(self):
         self.verify_bibxml_ids_creation(draft)
 
     def test_submit_update_individual(self):
-        IndividualDraftFactory(name='draft-ietf-random-thing', states=[('draft','rfc')], other_aliases=['rfc9999',], pages=5)
+        IndividualDraftFactory(name='draft-ietf-random-thing', states=[('draft','active'),('draft-iesg','approved')], pages=5)
```

Comment:
This is suspicious - will review - I suspect an RFC should be being built here instead.

Reply:
No - it's correct - this reflects replaces only pointing to draft objects, and the test is otherwise adjusted to exercise the "can't replace an RFC" and "can't replace an approved draft".